File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        hosh = {}
        for num in nums:
            if num in hosh:
                hosh[num] += 1
            else:
                hosh[num] = 1
        arr = sorted(hosh, key=hosh.get, reverse=False)[-k:]
        return arr

File: Data Structures & Algorithms/missing-number/submission-0.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int ref = 0;
        for(int i = 0; i < nums.size() + 1; i++){
            ref ^= i;
        }

        for(int i =0 ; i < nums.size(); i++){
            ref ^= nums[i];
        }

        return ref;
    }
};
